src/Week1/Homework1.hs

```diff
@@ -33,12 +34,11 @@ doubleEveryOther list =
 
 -- 3
 sumDigits :: [Integer] -> Integer
-sumDigits list = foldr (+) 0 $ concat $ map toDigits list
+sumDigits = sum . (concatMap toDigits)
```

Comment:
`()` here are redundant. You can try to install `hlint` tool. And run it on files. It can recommend you different ways to improve code readability.